src/layouts/components/Editor/extensions/Image/Image.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from "@tiptap/extension-image"
+import { ReactNodeViewRenderer } from "@tiptap/react"
+
+import { ImageView } from "./ImageView"
+
+export const IsomerImage = Image.extend({
+  parseHTML() {
+    return [
+      {
+        tag: "img",
+      },
+    ]
+  },
```

Comment:
question: I think this would not be needed now, since we are not parsing for our own wrapper.

Reply:
8524e38b0ae96562f801ffbbecd9f318bb7468e4
forgot we overloading